### DISCUSSION TITLE
The problem lies in providing the proof for Student Developer Pack

### DISCUSSION THREAD

**Discussion by ihorkomakhwork** [community/community#56147] [GitHub Education]
### Hi everyone,

Hi everyone, I am a student at Kyiv National University of Trade and Economics, and I have been trying to apply for the GitHub Student Developer Pack. However, my application has been rejected more than 19 times with the same reason.

"Hey ******,
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 22, 2023 for @.
• You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
• Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

Once you have resolved the issues with your submission, visit us again and reapply.
Have questions? Join the GitHub Education Community for answers.
💖 The GitHub Education Team
"

My full name is written in Cyrillic in the original documents, and I have tried various ways to enter the "Full Name" field, as well as different documents such as my student ID card, electronic student ID card, and academic record book, but all in vain. I have also provided a description. What could be the problem?

_No response_

**Comment by Ntshangase** [comment]
Hi @ihorkomakhwork 

It looks like your application has been approved, you an check your application status here: [Click me](https://education.github.com/discount_requests/application)

**Comment by aniket-dev30** [comment]
I have also the same problem  applied for student developer pack and upload my valid id card but it ask that to apply through your device but camera is not clicking proper HD photo. What should I do? I am unable to upload documents proof for this.
